crossword-xml-uploader/src/main/scala/com/gu/crossword/crosswords/CrosswordStore.scala

```diff
@@ -40,4 +40,11 @@ trait CrosswordStore {
     s3Client.deleteObject(config.crosswordsBucketName, awsKey)
   }
 
+  def archiveFailedCrosswordXMLFile(config: Config, awsKey: String): Unit = {
+    val processingFailedBucketName = "crossword-failed-files"
+    s3Client.copyObject(config.crosswordsBucketName, awsKey, processingFailedBucketName, awsKey)
```

Comment:
you mentioned something in standup about these failed files being purged after X days.

my suggestion at the time was to prefix the files by month, to aid sorting in S3 UI without having to have lifecycle configuration, something like...
```suggestion
    s3Client.copyObject(config.crosswordsBucketName, awsKey, processingFailedBucketName, s"$year-M$monthNumber/$awsKey")
```

but where is the failed bucket defined (platform repo) and is the lifecycle config pre-existing etc.

Reply:
I've removed the retention policy on the failed bucket for now, as the feedback in standup suggested this was not the ideal behaviour.

I couldn't see any of this defined in Cloudformation, so the bucket has been added in AWS directly

Reply:
nice, is it worth prefixing as suggested in my comment, just to make sorting in the UI easier? (non-blocking)